localisation/00_zuwc_united_localisation.csv

```diff
@@ -2645,14 +2645,42 @@ GetInSociety_society; in [This.Society.GetName];;;;;;;;;;;;;x
 GetInSocietyYellow_society; in �Y[This.Society.GetName]�!;;;;;;;;;;;;;x
 GetSocietyMembers_society;Society members;;;;;;;;;;;;;x
 GetSocietyMembers_brethren;brethren;;Br�der;;;;;;;;;;;x
-GetSocietyWe_society;[This.Society.GetName];;;;;;;;;;;;;x
-GetSocietyWe_we;we;;wir;;;;;;;;;;;x
-wc_united_sacrifice_all_prisoners_decision;Perform a Mass Sacrifice;;Ein Massenopfer darbringen;;;;;;;;;;;x
-wc_united_sacrifice_all_prisoners_decision_desc;Sacrifice all your prisoners to [Root.Religion.GetHighGodName].;;Opfert all Eure Gefangenen [Root.Religion.GetHighGodName].;;;;;;;;;;;x
-GetHighwaymenType_dragon_hunters;dragon hunters;;Drachenj�ger;;;;;;;;;;;x
-GetHighwaymenType_monster_hunters;monster hunters;;Monsterj�ger;;;;;;;;;;;x
-GetHighwaymenType_witch_hunters;witch hunters;;Hexenj�ger;;;;;;;;;;;x
-GetHighwaymenType_highwaymen;highwaymen;;Vagabunden;;;;;;;;;;;x
+GetSocietyWe_we;we;;;;;;;;;;;;;x
+EVTTITLE_WCVRK_110;Till Death Do Us Part;;Bis das der Tod uns scheidet;;;;;;;;;;;x
+EVTDESC_WCVRK_110;Recently, the illness that has befallen your husband has been growing worse with each passing day. Knowing well that he might not last another week, you have sought many solutions, be it magical or alchemical, but nothing seems to help alleviate his condition. The grief over your inevitable loss overcomes you, and in desperation you cry out in hopes something or someone will hear your pleas. To your surprise, a shadowy figure emerges and makes you a promise that death will never  part you and your husband... for a price. He offers you an unimaginable power and eternal life in exchange for loyalty, and you realize this might be the only way to keep your husband at your side. Conflicting thoughts cross your mind, but ultimately realize that if death can part you both...;;;;;;;;;;;;;x
```

Comment:
Can this only happen to sick husband? What about sick wife?

Reply:
WCVRK_110 is event for wife, WCVRK_111 is event for husband. Both are intended recipients. The husband needs to fall ill, then the wife makes a deal with Gorak's spirit, as it was in lore.